model/src/w3wavemd.F90

```diff
@@ -2405,7 +2449,7 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
             ISPEC = 0 
              JSEA = IY 
 #endif
-#endif
+#endif ! W3_SMC ... 
```

Comment:
Some compilers have issues with commends after the ifdef, please remove the ! W3_SMC hopefully we can systematically add that at some point in the future when we can resolve compiler issues.

Reply:
done